.github/workflows/cd.yml

```diff
@@ -1,18 +1,10 @@
 name: cd
 on:
-  workflow_run:
-    workflows: ["ci"]
-    branches-ignore: ["*"]
-    types:
-      - completed
```

Comment:
1/5 that I like running the test when a tag gets created as there is something tag-specific code. Also, this behaviour now differs from the core plugins, which seems unnecessary to me,

Reply:
I will add the test pipeline on the plugin-cd part [here](https://github.com/mattermost/actions-workflows/blob/main/.github/workflows/community-plugin-cd.yml) so that is unified. 

This specific directive was used in the past for security purposes 😄

Reply:
https://github.com/mattermost/actions-workflows/pull/41
WDYT ?

Reply:
LGTM :+1: